analyzer/src/main/kotlin/managers/GoDep.kt

```diff
@@ -163,9 +163,12 @@ class GoDep(name: String, analyzerConfig: AnalyzerConfiguration, repoConfig: Rep
 
     private fun importLegacyManifest(lockfileName: String, workingDir: File, gopath: File) {
         if (lockfileName.isNotEmpty() && !File(workingDir, lockfileName).isFile &&
-                !analyzerConfig.allowDynamicVersions) {
-            throw IllegalArgumentException("No lockfile found in ${workingDir.invariantSeparatorsPath}, dependency " +
-                    "versions are unstable.")
+            !analyzerConfig.allowDynamicVersions
+        ) {
+            throw IllegalArgumentException(
+                "No lockfile found in ${workingDir.invariantSeparatorsPath}, dependency " +
+                        "versions are unstable."
```

Comment:
indent ok?

Reply:
I did rely on the IDE's auto-formatting, see my comment below.